spanner/api/Spanner.Samples.Tests/UpdateUsingDmlReturningAsyncTest.cs

```diff
@@ -31,11 +32,13 @@ public async Task TestUpdateUsingDmlReturningAsync()
         await _spannerFixture.RunWithTemporaryDatabaseAsync(async databaseId =>
         {
             await CreateTableAndInsertData(databaseId);
+            FillMarketingBudgetsAsync(300000, 300000, databaseId);
+
             var sample = new UpdateUsingDmlReturningAsyncSample();
             var updatedMarketingBudget = await sample.UpdateUsingDmlReturningAsync(_spannerFixture.ProjectId, _spannerFixture.InstanceId, databaseId);
 
             Assert.Single(updatedMarketingBudget);
-            Assert.Equal(300000, updatedMarketingBudget[0]);
+            Assert.Equal(600000, updatedMarketingBudget[0]);
```

Comment:
I am not understanding that when we are inserting first and second budget as 300000, why is this being compared with the value 600000?

Reply:
300000 is inserted value in MarketingBudget column.  Sample `sample.UpdateUsingDmlReturningAsync` updates MarketingBudget column to twice it's value, which we are validating.